_data/rules.yml

```diff
@@ -37,6 +40,12 @@ permitted:
 - description: This license provides an express grant of patent rights from the contributor to the recipient.
   label: Patent Grant
   tag: patent-grant
+- description: This software may be bundled together with any other software.
```

Comment:
this is described in the FAQ. (Sorry it's rather difficult to dumb it down to a few words...)